src/test/scala/uk/gov/hmrc/play/auth/microservice/filters/AuthorisationFilterSpec.scala

```diff
@@ -33,7 +35,9 @@ import scala.concurrent.Future
 
 class AuthorisationFilterSpec extends WordSpecLike with MatchersResults with ScalaFutures {
 
- 
+  implicit val system = ActorSystem()
+  implicit val materializer = ActorMaterializer()
```

Comment:
How and where are these implicits used in the tests? Since we will continue to maintain this library, we need to know the background for these changes.